static/scss/partials/main.scss

```diff
@@ -2376,22 +2376,31 @@ input.input-has-error {
   width: 100%;
   padding: $spacing-lg $spacing-md 0 $spacing-md;
   margin: 0 auto;
-}
 
-.messages > div {
-  background-color: var(--grey50);
-}
```

Comment:
I found this to be redundant whilst testing.

Reply:
Nice find.